### PR TITLE
Added Bio

### PR DESCRIPTION
**Pull request by burkesquires**
Please find my bio attached.

### CODE REVIEW

File: _submissions/round-12/0/2015-02-05-about-BurkeSquires.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+@@ -0,0 +1,16 @@
```
Comment (gvwilson) [inline _submissions/round-12/0/2015-02-05-about-BurkeSquires.md]: Whoops - what is this line doing here?

Comment (burkesquires) [inline _submissions/round-12/0/2015-02-05-about-BurkeSquires.md]: Perhaps that was from the bio that I copied.

Comment (gvwilson) [inline _submissions/round-12/0/2015-02-05-about-BurkeSquires.md]: Yup - this is where I nod sagely and pretend that the mistake in mine 
was deliberate :-)

**Comment by burkesquires** [comment]
Thanks Jedi Master!

On a side note...I understand you will be coming south to Bethesda in early April...looking forward to it.

Cheers,

Burke

**Comment by gvwilson** [comment]
So my calendar tells me :-)

**Pull request merged**